index.js

```diff
@@ -48,23 +42,17 @@ function expressPlugin (fastify, options, next) {
 
   function runConnect (req, reply, next) {
     if (this[kMiddlewares].length > 0) {
-      this[kExpress](req.raw, reply.raw, next)
+      this.express(req.raw, reply.raw, next)
     } else {
       next()
     }
   }
 
-  function runOnSend (req, reply, payload, next) {
-    reply.raw.removeHeader('x-powered-by')
```

Comment:
this doesn't work anyways (not `reply.removeHeader('x-powered-by')` either, fwiw)

Reply:
Oh, I saw this now. Maybe we should also add the disable at the top. Sorry for my misunderstanding.

Reply:
Using `disable` is the way to go IMO

Reply:
Done 👍